### PR TITLE
docs: added docs on 08-wasm options api

### PR DESCRIPTION
**Pull request by srdtrk**
## Description

closes: #5383 

### Commit Message / Changelog Entry

```text
docs: added docs on 08-wasm options api
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: docs/docs/03-light-clients/04-wasm/03-integration.md [hunk 1]
```diff
@@ -275,3 +275,57 @@ In order to use the `08-wasm` module chains are required to register the `WasmSn
 ## Pin byte codes at start
 
 Wasm byte codes should be pinned to the WasmVM cache on every application start, therefore [this code](https://github.com/cosmos/ibc-go/blob/0ed221f687ffce75984bc57402fd678e07aa6cc5/modules/light-clients/08-wasm/testing/simapp/app.go#L821-L826) should be placed in `NewSimApp` function in `app.go`.
+
+## 08-wasm options
+
+The `08-wasm` module comes with an options api inspired by the one in `x/wasm`.
+Currently the only option available is the `WithQueryPlugins` option, which allows to register custom query plugins for the `08-wasm` module. The use of this api is optional and it is only required if the chain wants to register custom query plugins for the `08-wasm` module.
```
[the remark this answers is not in the document]
Comment (crodriguezvega) [reply docs/docs/03-light-clients/04-wasm/03-integration.md]: Changed both. :)

**Review by crodriguezvega — APPROVED** [review]
Thank you, @srdtrk! I pushed some changes myself, and I agree with @DimitrisJim's comment about adding some more information about `myAcceptList`.

File: docs/docs/03-light-clients/04-wasm/03-integration.md [hunk 2]
```diff
@@ -275,3 +275,57 @@ In order to use the `08-wasm` module chains are required to register the `WasmSn
...
+
+### `WithQueryPlugins`
+
+By default, the `08-wasm` module does not support any queries. However, it is possible to register custom query plugins for [`QueryRequest::Custom`](https://github.com/CosmWasm/cosmwasm/blob/v1.5.0/packages/std/src/query/mod.rs#L45) and [`QueryRequest::Stargate`](https://github.com/CosmWasm/cosmwasm/blob/v1.5.0/packages/std/src/query/mod.rs#L54-L61).
+
+Assuming that the keeper is not yet instantiated, the following sample code shows how to register query plugins for the `08-wasm` module.
+
+We first construct a [`QueryPlugins`](https://github.com/cosmos/ibc-go/blob/08-wasm/release/v0.1.x%2Bibc-go-v8.0.x-wasmvm-v1.5.x/modules/light-clients/08-wasm/types/querier.go#L78-L87) object with the desired query plugins:
```
[the remark this answers is not in the document]
Comment (damiannolan) [reply docs/docs/03-light-clients/04-wasm/03-integration.md:261]: we can update it to use a final tag once cut

**Review by damiannolan — APPROVED** [review]
Awesome again @srdtrk! Thanks!

File: docs/docs/03-light-clients/04-wasm/03-integration.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (srdtrk) [reply docs/docs/03-light-clients/04-wasm/03-integration.md:261]: cannot use relative paths outside the docs directory in docusaurus

**Pull request merged**